ansible/README.md

```diff
@@ -0,0 +1,178 @@
+# README
+
+To run this playbook, you should have Ansible installed locally.  On a
+Debian system, you can do this with:
+
+    $ sudo apt-get install ansible
+
+This will create a directory `/etc/ansible` and within that a file
+`/etc/ansible/hosts`.  To run this playbook, you should add the
+following lines to the `hosts` file:
+
+```
+[mediawiki]
+YOUR_SERVER_NAME_OR_IP_HERE
+```
```

Comment:
We should mention that this step isn't needed for the vagrant/virtualbox setup.  Or maybe better to introduce the different options here: with vagrant/virtualbox and without vagrant/virtualbox, like with separate headings or something.